packages/pipeline-editor/src/CustomFormControls/CustomFieldTemplate.tsx

```diff
@@ -33,8 +33,10 @@ export const CustomFieldTemplate: React.FC<FieldTemplateProps> = (props) => {
   return (
     <div
       className={`${props.classNames} ${
-        props.schema.oneOf ? "field-oneOf" : ""
-      }`}
+        props.schema.properties?.value &&
+        props.schema.properties?.widget &&
+        "small-object-field"
```

Comment:
this seems like it would result in a class name of `true` not `small-object-field` as I would assume is expected

Reply:
No I think actually it will / did work - it's a typescript thing, couldn't find documentation for it but it just saves a line of code